assets/designer/environments/designer-cv-transform/context/conda_dependencies.yaml

```diff
@@ -0,0 +1,30 @@
+name: project_environment
+channels:
+  - defaults
```

Comment:
I'd recommend to put conda-forge channel first. Channel is well maintained, defaults (anaconda) is usually significantly out of date